### PR TITLE
users: Document advanced config option remoteGUIport.

### PR DESCRIPTION
**Pull request by acolomb**
Implemented in https://github.com/syncthing/syncthing/pull/7017.

### CODE REVIEW

**Comment by tomasz1986** [comment]
Just for the record, this is going to be available from v1.12.1.

File: users/config.rst [hunk 1]
```diff
@@ -81,6 +81,7 @@ The following shows an example of the default configuration file (IDs will diffe
             <maxSendKbps>0</maxSendKbps>
             <maxRecvKbps>0</maxRecvKbps>
             <maxRequestKiB>0</maxRequestKiB>
+            <remoteGUIPort>8384</remoteGUIPort>
```
Comment (tomasz1986) [inline users/config.rst:84]: The value should be

```
<remoteGUIPort>0</remoteGUIPort>
```

here, as this is supposed to show the default configuration. The default value is `0`.

File: users/config.rst [hunk 2]
```diff
@@ -408,6 +409,7 @@ Device Element
         <maxSendKbps>0</maxSendKbps>
         <maxRecvKbps>0</maxRecvKbps>
         <maxRequestKiB>0</maxRequestKiB>
+        <remoteGUIPort>8384</remoteGUIPort>
```
Comment (tomasz1986) [inline users/config.rst:412]: I would change this to `<remoteGUIPort>0</remoteGUIPort>`, but also add `<remoteGUIPort>8384</remoteGUIPort>` to the next device config below. This way we have both the default and a custom values shown.

**Comment by acolomb** [comment]
Thanks @tomasz1986 for the review.  Both points changed as suggested.

As v1.12.1 will be tagged in a couple of days, this should be merged before then, right?

**Comment by tomasz1986** [comment]
> As v1.12.1 will be tagged in a couple of days, this should be merged before then, right?

The setting is already in the RC, so I would say yes 🙂.

**Review by tomasz1986 — APPROVED** [review]

**Pull request merged**